### PR TITLE
documented fluttertexture.h

### PR DESCRIPTION
**Pull request by gaaclarke**

### CODE REVIEW

File: shell/platform/darwin/common/framework/Headers/FlutterTexture.h [hunk 1]
```diff
@@ -26,9 +32,26 @@ FLUTTER_EXPORT
...
 @protocol FlutterTextureRegistry <NSObject>
+/**
+ * Registers a `FlutterTexture` for usage in Flutter and returns an id that can be used to reference
+ * that texture when calling into Flutter with channels.
+ */
 - (int64_t)registerTexture:(NSObject<FlutterTexture>*)texture;
+/**
+ * Notifies Flutter that the content of the previously registered texture has been updated.
+ *
+ * See also: `flutter::PlatformView::MarkTextureFrameAvailable`
```
Comment (chinmaygarde) [inline shell/platform/darwin/common/framework/Headers/FlutterTexture.h]: This is an internal method that users of this header will not have access to.

Comment (chinmaygarde) [reply shell/platform/darwin/common/framework/Headers/FlutterTexture.h]: Maybe also mention that in response to this call, the engine will attempt to resolve the pixel buffer via the `copyPixelBuffer` call on the render thread.

Comment (gaaclarke) [reply shell/platform/darwin/common/framework/Headers/FlutterTexture.h]: Render thread?  Gpu thread, right?
Also, this call is meant to happen on the platform thread, right?

Comment (gaaclarke) [reply shell/platform/darwin/common/framework/Headers/FlutterTexture.h]: Updated.

Comment (chinmaygarde) [reply shell/platform/darwin/common/framework/Headers/FlutterTexture.h]: This call should happen on the platform thread but the pixel buffer will be copied on the GPU thread. Sorry about the mixed terminology, still processing https://github.com/flutter/flutter/issues/29443.

**Comment by gaaclarke** [comment]
landing without build_and_test_windows_unopt_debug, after 52 minutes still hasn't converged.

**Pull request merged**